common/graphics/persistent_cache.cc

```diff
@@ -191,7 +192,35 @@ sk_sp<SkData> ParseBase64(const std::string& input) {
   return data;
 }
 
-std::vector<PersistentCache::SkSLCache> PersistentCache::LoadSkSLs() {
+size_t PersistentCache::PrecompileKnownSKSLs(GrDirectContext* context) const {
```

Comment:
In Skia and elsewhere in the engine the spelling is "SkSL".  I would prefer using that spelling in this PR for consistency.